design/snapshots.md

```diff
@@ -142,7 +142,7 @@ that applies the contents of the layer to target location and calculates the
 DiffID of the unpacked layer (this is a requirement for docker
 implementation):
 
-	digest, err := unpackLayer(tmpLocation, layer) // unpack into layer location
+	digest, err := unpackLayer(tmpLocation, layerPath) // unpack into layer location
```

Comment:
This was correct. The `layer` here is the `io.Reader` of the actual layer content.